app/modules/sightings/models.py

```diff
@@ -538,198 +543,137 @@ def _get_pipeline_status_detection(self):
-        status['inProgress'] = False
-        status['complete'] = True
-        status['end'] = self.created.isoformat() + 'Z'
-        status['progress'] = 1.0
+        # leaving this for prosperity and consideration, but going to comment out for now
+        #
+        # The curation stage starts when manual annotation OR detection adds the first annotation to the asset group sighting.
+        # annotations = self.get_annotations()
+        # if annotations and len(annotations) > 1:
+        #    times = [ann.created for ann in annotations]
+        #    first_time = min(times)
+        #    status['start'] = first_time.isoformat() + 'Z'
+        # else:
+        #    status['start'] = self.created.isoformat() + 'Z'
         return status
 
     def _get_pipeline_status_identification(self):
-        from app.utils import datetime_string_to_isoformat
+        from app.modules.progress.models import ProgressStatus
+
+        progress = self.progress_identification
+        # no progress object, we assume it has not yet started
+        if not progress:
+            status = {
+                'skipped': False,
+                'inProgress': False,
+                'failed': False,
+                'complete': False,
+                'message': 'missing Progress',
+                'steps': 0,
+                'stepsComplete': 0,
+                'progress': 0,
+                'start': None,
+                'end': None,
+                'eta': None,
+                'ahead': None,
+                'status': None,
+                'description': None,
+            }
+            # if migrated, should look skipped rather than not started
+            if self.is_migrated_data():
+                status['skipped'] = True
+            return status
 
-        annots = self.get_annotations()
         status = {
+            # start with these false and set below
             'skipped': False,
-            'start': None,
-            'end': None,
             'inProgress': False,
-            'complete': False,
             'failed': False,
-            'message': None,
-            'eta': None,
-            'ahead': None,
-            'jobs': None,
-            'numJobs': None,
-            'numJobsActive': None,
-            'numJobsFailed': None,
-            # these are only based on 0th id_config (all that we have for mvp)
-            'matchingSetQueryPassed': None,
-            'matchingSetQueryUsed': None,
-            # this is only based on 0th annotation (so is approximate)
-            # 'matchingSetSize': None,   # dropped now cuz its expensive
-            'numAttempts': None,
-            'numAttemptsMax': None,
-            'idConfigs': None,
-            'numAssets': len(self.get_encounter_assets()),
-            'numAnnotations': len(annots),
-            'steps': 0,
-            'stepsComplete': 0,
-            'progress': None,
+            'complete': False,
+            'message': progress.message,
```

Comment:
Isn't this in the if not progress block or am I misreading the code?

Reply:
i think thats just a git/diff weirdness.  i am looking at the actual file, and line 650 is _outside_ the `if not progress`:

```
        if not progress:
            status = {
                ...
                'description': None,
            }
            # if migrated, should look skipped rather than not started
            if self.is_migrated_data():
                status['skipped'] = True
            return status

        status = {
            # start with these false and set below
            'skipped': False,
            'inProgress': False,
            'failed': False,
            'complete': False,
 >>>        'message': progress.message,
```